lib/librato/rails/subscribers/sql.rb

```diff
@@ -4,21 +4,21 @@ module Subscribers
 
       # SQL
 
-      ActiveSupport::Notifications.subscribe 'sql.active_record' do |*args|
-        payload = args.last
+      ActiveSupport::Notifications.subscribe "sql.active_record" do |*args|
+
+        event = ActiveSupport::Notifications::Event.new(*args)
```

Comment:
Not sure if we actually need the extra object instantiation here, this can get called a ton.